packages/driver/cypress/e2e/e2e/origin/commands/spies_stubs_clocks.cy.ts

```diff
@@ -150,7 +151,7 @@ context('cy.origin spies, stubs, and clock', { browser: '!webkit' }, () => {
 
         expect(clockLog.name).to.equal('clock')
 
-        const consoleProps = clockLog.consoleProps()
```

Comment:
Why is `consoleProps` no longer a function?

Reply:
This test wasn't super solid.Running without my changes, I have seen this fail with the following error:
<img width="819" alt="Screen Shot 2023-12-19 at 4 20 16 PM" src="https://github.com/cypress-io/cypress/assets/14099737/c02a0924-aa92-4c21-bac3-58f5e7cedbd0">


Def observed in Firefox.

When a log is created, we wrap it to inject a few console log properties. When the secondary origin sends the log update, the console props are sent as an object and it is not re-wrapped so the consoleProp attributes are always an object. You can see on line 177 where we had attempted to account for this by checking if it was a func or not.